agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4315,8 +4315,32 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
     }
 
     for (auto tlvWscM2 : m2_list) {
+        auto encrypted_settings = tlvWscM2->encrypted_settings();
+        auto ssid  = std::string(encrypted_settings->ssid(), encrypted_settings->ssid_length());
+        auto bssid = network_utils::mac_to_string(encrypted_settings->bssid_attr().data);
+        auto authentication_type = encrypted_settings->authentication_type_attr().data;
+        auto encryption_type     = encrypted_settings->encryption_type_attr().data;
+        // char network_key[WSC::WSC_MAX_NETWORK_KEY_LENGTH];
+        // std::copy_n(encrypted_settings.network_key_attr().data,
```

Comment:
dead code?

Reply:
Not dead, should be a TODO here. Will add one.